support-frontend/assets/pages/[countryGroupId]/components/guardianLightCards.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ProductDescription } from 'helpers/productCatalog';
+
+export type GuardianLightCardsProps = {
+	cardsContent: Array<{
+		link: string;
+		productDescription: ProductDescription;
+		ctaCopy: string;
+	}>;
+};
+
+export function GuardianLightCards({
+	cardsContent,
+}: GuardianLightCardsProps): JSX.Element {
+	return (
+		<div>
+			GuardianLightCards {cardsContent[0].productDescription.label}{' '}
+			{cardsContent[1].productDescription.label}
```

Comment:
I think this is just a placeholder so definitely no big deal, but I'm surprised TypeScript isn't complaining about this. Since its an array I think the items at indexes 0 and 1 are possibly undefined?

Reply:
Not sure I understand this. I believe both cards are defined which must contain a `productDescription` which must contain a `label`. Yes this is a placeholder anyway and will be replaced with this [PR](https://github.com/guardian/support-frontend/pull/6577). If I have mis-understood I should be able to correct in that PR.